zstacklib/zstacklib/gpu/vendors/vastai.py

```
@@ -0,0 +1,360 @@
+                show_data = json.loads(output.decode('utf-8').strip())
+            else:
+                show_data = output
+        except (ValueError, TypeError) as e:
+            logger.error("Failed to parse VastAI show JSON: %s" % str(e))
+            return results
+        
+        if not show_data or "elem" not in show_data:
+            return results
+        
+        # Get GPU type to determine field names
+        gpu_type = cls.get_vastai_type()
```

Ruff (0.14.14)

Local variable `gpu_type` is assigned to but never used

Remove assignment to unused variable `gpu_type`

(F841)